index.md

```diff
@@ -1514,16 +1148,13 @@ def hello():
 
 if __name__ == '__main__':
     hello()
-
-
 ```
 
+
 ### E9991: Dynamic Execution {#E9991}
 
 This error occurs when you use a dynamic execution of code. Use of builtin
 functions exec(), eval() and compile() is not allowed.
 
-```python
-for i in range(0, 2):
-    num = eval('i + 1')   # Error on this line
-```
+~~~~ {.python include="dynamic_execution_checker_example.py"}
```

Comment:
@nigef The file has been renamed to `dynamic_execution_example.py` in the pyta repo.